primitives/core/src/assertion.rs

```diff
@@ -246,6 +246,9 @@ pub enum Assertion {
 	#[codec(index = 19)]
 	WeirdoGhostGangHolder,
 
+	#[codec(index = 20)]
+	BRC20AmountHolder,
+
```

Comment:
As we discussed earlier, `LIT staking` ' index is 20, `index = 23` is available for `BRCAmountHolder`